test/test_image.py

```diff
@@ -68,7 +102,7 @@ def test_damaged_images(self):
                 decode_jpeg(data)
 
     def test_encode_jpeg(self):
-        for img_path in get_images(IMAGE_ROOT, ".jpg"):
+        for img_path in get_images(ENCODE_JPEG, ".jpg"):
```

Comment:
This test is problematic. Once it detects a jpg it assumes it can write to specific locations. This assumption is problematic so I had to move its assets to a separate folder. It's worth providing a more permanent fix for this unit-test on the future.

Reply:
Should be handled by #3002